chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationCriterion.java

```diff
@@ -41,9 +47,6 @@
     @XmlElement(required = true, nillable = false)
     private Boolean removed;
 
-    public CertificationCriterion() {
```

Comment:
Isn't this needed for the schemagen thing?

Reply:
Based on what I said last week - Yes.  But I was getting maven build errors when it was missing.  For some reason, this class was not throwing an error.  Anyway, I added it back in just in case.